packages/table/src/table.tsx

```diff
@@ -1004,6 +1092,53 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         this.handleSelection([]);
     }
 
+    private handleFocusMove = (e: KeyboardEvent, direction: "up" | "down" | "left" | "right") => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        const {focusedCellCoordinates} = this.state;
```

Comment:
worth adding to the `whitespace` rule, yeah. we check import statements but not destructuring